### DISCUSSION TITLE
Related to the blog post of speed estimation using supervision

### DISCUSSION THREAD

**Discussion by jasminaaa20** [roboflow/notebooks#247] [Q&A] [answered]
Hi Piotr. In the blog post of the speed estimation problem, you have stated that the slope of the road is a hidden complexity. I don't agree with that.
To my knowledge, the 50m markings on the sides of the road are planted for every 50 meters the road goes, not the horizontal projection of the road.
Therefore, when we take the distance the car has traveled based on the markings on the sides of the road, the slope won't be an issue.

I could be wrong. Please be kind enough to reply

**Comment by SkalskiP** [comment, marked as answer]
Hello 👋🏻 And if I make more precise and say "changing slope"? Will you still disagree. If the answers is yes than I'm happy to create some visualizations.

**Comment by jasminaaa20** [comment]
By "changing slope", if you meant a curve, I would agree. But then we won't be able to specify an angle as you have mentioned in the blog. We would have to calculate the equation of the curve, which is practically complex.

**Comment by SkalskiP** [comment]
Converting this issue into discussion and puttinng into Q&A section.
